src/app/modules/side-menu/components/filter-menu/filter-menu.component.ts

```diff
@@ -58,52 +71,78 @@ export class FilterMenuComponent extends NgDestoryBase implements OnInit {
   isIosOrAndroid: boolean;
   isMobileWeb: boolean;
   showObservationTypes = false;
+  showCompetances = false;
   isIndeterminate: boolean;
   masterCheck: boolean;
   nickName: string;
 
   observationTypesOptions: ObservationTypeView[];
 
+  selectedVal: string;
+  competenceOptions: CompetenceItem[];
+  automaticStation: CompetenceItem;
+  isShowAutomaticStation = false;
+  isAutomaticStationChecked = true;
+  chosenValue = 'all';
   constructor(
     private platform: Platform,
     private userSettingService: UserSettingService,
     private searchCriteriaService: SearchCriteriaService,
+    private searchService: SearchService,
     private cdr: ChangeDetectorRef,
     private kdv: KdvService,
     private logger: LoggingService
   ) {
     super();
   }
+  /*
+    ukjent 0, 100
+    * 110
```

Comment:
Her kan vi legge inn verdiene for de andre modusene også. For dette er vel bare for snø?